### DISCUSSION TITLE
Disney Classics Collection is being updated November 9th to include Jungle Book.

### DISCUSSION THREAD

**Discussion by retroguy42** [farmerbb/RED-Project#25] [General]
I'm just mentioning this so that the instructions can be updated appropriately. The Jungle Book will be a $10 DLC for people who already own the collection. The price for the collection is being changed to $30 USD since it will now have The Jungle Book.

**Comment by farmerbb** [comment]
Thanks for putting this on my radar 👍

**Comment by retroguy42** [comment]
Steam and GoG have the new content now. Steam adds on Capcom's Aladdin on SNES and The Jungle Book via DLC. GoG just removed the old version from the store and gave previous owners a 66% discount (so no DLC, just $30 for the whole package).
